ksim/utils/mujoco.py

```diff
@@ -168,21 +168,55 @@ def get_floor_idx(physics_model: PhysicsModel, floor_name: str = "floor") -> int
 
 
 def geoms_colliding(state: PhysicsData, geom1: Array, geom2: Array) -> Array:
```

Comment:
i'm not sure what the new version of this function is doing that the old version was not doing...